api/handlers.go

```diff
@@ -557,6 +559,122 @@ func (si *ServerImplementation) LookupApplicationByID(ctx echo.Context, applicat
 	})
 }
 
+// LookupApplicationBoxByIDAndName returns the value of an application's box
+// (GET /v2/applications/{application-id}/box)
+func (si *ServerImplementation) LookupApplicationBoxByIDAndName(ctx echo.Context, applicationID uint64, params generated.LookupApplicationBoxByIDAndNameParams) error {
```

Comment:
please add a check asserting that parameters of type uint64 is <= math.MaxInt64 in all handler methods. this is because indexer db can only store values up to max int64. See existing handlers for examples.

Reply:
It turns out that this guard was already in place (probably due to work that you recently put in). By adding 4 new test cases to the fixtures test, we can verify that this is the case.

See the most recent commit.

Reply:
Upon further investigation, I'm no longer as confident as I was previously that the problem has been addressed. Will keep this thread open and update you as I learn more.

Reply:
Per [this commit](https://github.com/algorand/indexer/pull/1117/commits/b5025ad640fabac0d778b4cac60d558a698ed560) I'm conforming to the same response code and error as with other indexer treatments.

Resolving

Reply:
Unresolving temporarily, as the commit above shows the convenience of having `fixtures_test.go` lying around.